src/app/products/manage-delinquency-buckets/delinquency-bucket/create-bucket/create-bucket.component.ts

```
@@ -158,30 +200,45 @@ export class CreateBucketComponent implements OnInit {
+        ranges: ranges
+      };
+    } else if (this.isWorkingCapitalBucket) {
+      const payload = this.bucketForm.value;
+      const bucketName = payload['name'];
+      return {
+        bucketType: bucketType,
+        name: bucketName,
+        minimumPaymentPeriodAndRule: payload
+      };
+    }
+  }
```

Biome (2.4.7)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)